app/views/layouts/application.html.erb

```diff
@@ -34,6 +34,7 @@
         full_path: request.fullpath
       ).inspect.html_safe %>);
     </script>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```

Comment:
Renders a `<script>` tag for `/packs/js/application.js` which is the entry-point for the front-end portion of this implementation.